std_msgs/Header header
std_msgs/UInt16 total_points
std_msgs/UInt16MultiArray point_idx
geometry_msgs/Point[] points
std_msgs/Float32MultiArray velocity
std_msgs/Float32MultiArray snr